lib/openai.rb

```diff
@@ -17,7 +18,7 @@ class Configuration
 
     DEFAULT_API_VERSION = "v1".freeze
     DEFAULT_URI_BASE = "https://api.openai.com/".freeze
-    DEFAULT_REQUEST_TIMEOUT = 120
+    DEFAULT_REQUEST_TIMEOUT = 0 # Never time out.
```

Comment:
given how unstable the OpenAI API is, this doesn't seem a great default... I personally would prefer a much shorter than 120 seconds request timeout.

Reply:
Agree with @bf4. 120 seconds seems fine to me though